site/en/docs/extensions/reference/storage/index.md

```diff
@@ -13,76 +13,65 @@ The Storage API provides an extension-specific way to persist user data and stat
 - Stored settings persist even when using [split incognito][incognito].
 - Includes an exclusive read-only [managed storage area][manifest-storage] for enterprise policies.
 
+## Manifest {: #manifest}
+
+To use the storage API, declare the `"storage"` permission in the extension
+[manifest][doc-manifest]. For example:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "storage"
+  ],
+  ...
+}
+```
+
+## Concepts and usage {: #usage }
+
 {% Details %}
 {% DetailsSummary %}
 Can extensions use web storage APIs?
 {% endDetailsSummary %}
 
-Even though extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+While extensions can use the [`Storage`][mdn-storage] interface (accessible from `window.localStorage`) in some contexts (popup and other HTML pages), we don't recommend it for the following reasons:
 
-- Extension's service worker cannot access `Storage`.
+- Extension's service worker can't access `Storage`.
 - Content scripts share storage with the host page.
 - Data saved using the `Storage` interface is lost when the user clears their browsing history.
 
 To move data from web storage APIs to extension storage APIs from a service worker:
 
 1. Create an offscreen document with a conversion routine and an [`onMessage`][on-message] handler.
-1. Add a conversion routine to an offscreen document.
-1. In the extension service worker check `chrome.storage` for your data.
-1. If your data isn't found, [create][create-offscreen] an offscreen document and call [`sendMessage()`][send-message] to start the conversion routine.
-1. Inside the offscreen document's `onMessage` handler, call the conversion routine.
+2. Add a conversion routine to an offscreen document. <!--Is 2 an alternative to 1? If so, I'd like to combine them and present them as options.-->
+3. In the extension service worker, check `chrome.storage` for your data.
+4. If your data isn't found, [create][create-offscreen] an offscreen document and call [`sendMessage()`][send-message] to start the conversion routine.
+5. Inside the offscreen document's `onMessage` handler, call the conversion routine.
```

Comment:
```suggestion
1. In the extension service worker, check `chrome.storage` for your data.
1. If your data isn't found, [create][create-offscreen] an offscreen document and call [`sendMessage()`][send-message] to start the conversion routine.
1. Inside the offscreen document's `onMessage` handler, call the conversion routine.
```

Reply:
I rewrote the first step a while back and neglected to remove the old version.

FYI, the markdown converter numbers automatically when you use '1' for every step. That's what we always do so that we don't need to renumber everything when we change a procedure.